static/js/components/LearnerInfoCard.js

```diff
@@ -18,50 +22,85 @@ export default class LearnerInfoCard extends React.Component {
     profile: Profile,
     toggleShowPersonalDialog: () => void,
     toggleShowAboutMeDialog: () => void,
+    openLearnerEmailComposer: () => void,
   };
 
-  email = (email: string): React$Element<*> => (
-    <span className="profile-email">{email}</span>
-  );
+  ownProfilePage: boolean;
+
+  isOwnProfilePage = (): boolean => {
+    if (R.isNil(this.ownProfilePage)) {
+      this.ownProfilePage = isProfileOfLoggedinUser(this.props.profile);
+    }
+    return this.ownProfilePage;
+  };
```

Comment:
@noisecapella i'm kinda in between on this. not the cleanest pattern, but it avoids the unnecessary repeated calculation of the same conditions